.github/workflows/ci.yml

```diff
@@ -40,7 +40,7 @@ env:
   # Once 1.76 coes out, we can use that instead of nightly.
   DOCS_RUST_VERSION: "nightly-2023-12-17"
   # This is the MSRV used by `wgpu` itself and all surrounding infrastructure.
-  REPO_MSRV: "1.71"
+  REPO_MSRV: "1.76"
```

Comment:
Note that Gecko currently is on rustc 1.74

Reply:
That's fine, everything gecko depends on should be checked by the other msrv check which is still at 1.70